lib/index.js

```diff
@@ -34,7 +34,8 @@ module.exports = strapi => {
       const client = algoliasearch(applicationId, apiKey);
 
       const initIndex = (indexName) => {
-        return client.initIndex(`${prefix}_${indexName}`);
+        const index = (prefix && `${prefix}_${indexName}`) || `${indexName}`;
```

Comment:
Template string here is redundant.

```suggestion
        const index = (prefix && `${prefix}_${indexName}`) || indexName;
```

As well as `prefix` should be strictly equal to `false`.
Something like this
```js
return client.initIndex((prefix === false) ? indexName : `${prefix}_${indexName}`);
```